utils/decorator.py

```diff
@@ -0,0 +1,24 @@
+import jwt
+import json
+import requests
+
+from django.http           import JsonResponse
+from users.models          import User
+
+import my_settings
```

Comment:
```suggestion
from django.http           import JsonResponse

from users.models          import User
import my_settings
```

1. 파이썬 모듈
2. 장고 모듈
3. 내가 만든 모듈 및 클래스

임포트 구분해주세요!